### PR TITLE
Added temporary fix for categories

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**
- Read category(brand id) from bid ext. If no mapping take the fist category from bid if there are some
- Unit test temp fix

### CODE REVIEW

File: adapters/appnexus/appnexus.go [hunk 1]
```diff
@@ -486,6 +488,9 @@ func getMediaTypeForBid(bid *appnexusBidExt) (openrtb_ext.BidType, error) {
 // getIabCategoryForBid maps an appnexus brand id to an IAB category.
 func (a *AppNexusAdapter) getIabCategoryForBid(bid *appnexusBidExt) (string, error) {
 	// TODO: Change from BrandId to a CategoryId once that is returned from impbus
+	if &bid.Appnexus == nil || bid.Appnexus.BrandId == 0 {
```
Comment (jmaynardxandr) [inline adapters/appnexus/appnexus.go]: Should be `if bid == nil || bid.Appnexus.BrandId == 0 {` as long as bid is valid Appnexus field will always exist even if it's initialized with all zero values.

Comment (VeronikaSolovei9) [reply adapters/appnexus/appnexus.go]: Done

**Review by jmaynardxandr — APPROVED** [review]

**Comment by jmaynardxandr** [comment]
Please make sure to merge into develop branch instead of master so it doesn't effect the open PR into the upstream repo. Thanks!

**Comment by jmaynardxandr** [comment]
This temp fix is no longer needed with latest changes in new_simlified_endpoint branch to handle the correct category id in appnexus adapter.

**Pull request closed without merging**